bin/dap_test

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
I've been thinking that `bin/integration_tests` doesn't follow the naming standard we'd like to move towards (`bin/test_unit`, `bin/test_integration`, `bin/test_e2e`, etc) and should be renamed. I'd like to not add to this debt here, and I also wonder if this shouldn't just be built into the `bin/test_integration` script with an `--enterprise` flag?

Reply:
I agree that the dap tests should be integrated into the current test scripts. However I just took a look and I think it will require a pretty deep rework of the `integration_test` script to make it clean. This PR has been open for over a month so I would rather add that to a new tech debt issue where I can clean up all of the technical debt relating to these scripts. What do you think?

Reply:
that sounds fine - please file an issue now and link it here, though.

Reply:
https://github.com/cyberark/conjur-openapi-spec/issues/166